src/SFA.DAS.AssessorService.Web/Controllers/Apply/ApplicationController.cs

```diff
@@ -734,6 +739,21 @@ public async Task<IActionResult> Feedback(Guid id)
             return View("~/Views/Application/Feedback.cshtml", sequenceVm);
         }
 
+        private ApplicationSubmissionType GetApplicationSubmissionTypeAsync(ApplicationResponse application, Sequence sequence)
```

Comment:
There is already an ApplicationTypes class which has definitions for types of application StandardWithdrawal etc - so we should not have something new for that.

Other views e.g. 'Feedback.cshtml' are just using the ApplyConst directly - so that would be more consistent if you put the SequenceNo in the view model and the test against the ApplyConst which is also used in dozens of places, so it needs to kept consistent

Reply:
updated